projects/blastex/index.js

```diff
@@ -1,13 +1,12 @@
 const { sumTokensExport } = require('../helper/unknownTokens')
 
-const main = '0xdC0C0746CA0954E6C2284D1a97cC85474B051EbB'
-const token = '0x800ce05BaDE6B87E1552d0301Ea3393ccFf42F4A'
-const lps = ['0x974D36201171cA8D28Cc3F46972349d24Be3A303']
+const main = '0x8C1DBB14c012bCCdB3477bc1625A3DCfD0F61ac2'
+const token = '0x5C598E410De1214D77EBB166102471065E7b2596'
+const lps = ['0xdaE375F817B465f3a226284Af0Ad5Fa2387274EA']
 
 module.exports = {
   blast: {
-    tvl: () => ({}),
-    pool2: sumTokensExport({ owner: main, tokens: lps, useDefaultCoreAssets: true }),
+    tvl: sumTokensExport({ owner: main, tokens: lps, useDefaultCoreAssets: true }),
```

Comment:
revert this change, back to counting staked LP as pool2

Reply:
Done.